content/authors/jenny-wong/_index.md

```diff
@@ -10,7 +10,7 @@ authors:
 superuser: false
 
 # Role/position (e.g., Professor of Artificial Intelligence)
-role: Product Manager
+role: Open Source Applications Engineer
```

Comment:
Woah I didn't sign off on this?! There can only be one cowboy in these parts :cowboy_hat_face: 

:partying_face: